src/main/java/com/chriscarini/jetbrains/locchangecountdetector/LoCService.java

```diff
@@ -148,43 +73,57 @@ public LoCService(@NotNull final Project project) {
             }
 
             loc = additions + deletions;
-        } catch (IOException | InterruptedException ex) {
-            ex.printStackTrace();
+        } catch (VcsException e) {
+            e.printStackTrace();
         }
 
         return new Pair<>(loc, filesChanged);
     }
 
-    private static Pair<Integer, Integer> getGitDiffNumstat(@NotNull final Path path) {
-        final ProcessBuilder processBuilder2 = new ProcessBuilder();
-        processBuilder2.command("bash", "-c", "git diff HEAD --numstat").directory(path.toFile());
+    private Pair<Integer, Integer> getGitDiffNumstat(@NotNull final Path path) {
 
         int filesChanged = 0;
         int additions = 0;
         int deletions = 0;
 
         try {
-            Process process = processBuilder2.start();
-            StringBuilder output = new StringBuilder();
-            BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
-
-            String line;
-            while ((line = reader.readLine()) != null) {
-                final String[] lineSplit = line.split("\t");
-                if (lineSplit.length == 0 || Objects.equals(lineSplit[0], "")) {
-                    continue;
-                }
-                filesChanged += 1;
-                additions += Integer.parseInt(lineSplit[0]);
-                deletions += Integer.parseInt(lineSplit[1]);
+            List<VirtualFile> virtualFiles = new ArrayList<>();
+            String line = getDiffNumStat(this.project, this.project.getBasePath(), virtualFiles);
+
+            final String[] lineSplit = line.split("\t");
+            if (lineSplit.length == 0 || Objects.equals(lineSplit[0], "")) {
+                return new Pair<>(additions + deletions, filesChanged);
```

Comment:
IJ telling me `additions + deletions` is always zero here; if that's true, let's just explicitly change this to say so:
```suggestion
                return new Pair<>(0, filesChanged);
```

Reply:
self-resolved